include/deal.II/base/data_out_base.h

```diff
@@ -2141,7 +2310,8 @@ class DataOutInterface
    * performance on parallel filesystems. Also see
    * DataOutInterface::write_vtu().
    */
-  void write_vtu_in_parallel (const char *filename, MPI_Comm comm) const;
+  void write_vtu_in_parallel (const char *filename,
+                              MPI_Comm comm) const;
 
   /**
    * Some visualization programs, such as ParaView, can read several separate
```

Comment:
can you add something like ".. master record, by calling DataOutBase::write_pvtu_record" or something similar at the end of this paragraph?